PAWS/PAWS.swift

```diff
@@ -10,22 +10,41 @@ import CardinalKit
 import PAWSOnboardingFlow
 import PAWSSharedContext
 import SwiftUI
-
+import PAWSLandingScreen
 
 @main
 struct PAWS: App {
     @UIApplicationDelegateAdaptor(PAWSAppDelegate.self) var appDelegate
     @AppStorage(StorageKeys.onboardingFlowComplete) var completedOnboardingFlow = false
+//    @AppStorage(StorageKeys.accountCreated) var completedAccountSetup = false
+
+    @State var pressedStart = false
     
+    var isSheetPresented: Binding<Bool> {
+        Binding(
+            get: {
+                !completedOnboardingFlow && pressedStart
+            }, set: { _ in }
+        )
+    }
     
     var body: some Scene {
         WindowGroup {
-            HomeView()
-                .sheet(isPresented: !$completedOnboardingFlow) {
-                    OnboardingFlow()
+//            if  completedAccountSetup {
+            Group {
+                if completedOnboardingFlow  {
+                    HomeView()
+                } else {
+                    LandingScreen(pressedStart: $pressedStart)
                 }
-                .testingSetup()
+            }
+                    .sheet(isPresented: isSheetPresented) {
+                        OnboardingFlow()
+                    }
+                    .interactiveDismissDisabled(true)
+                    .testingSetup()
```

Comment:
Please indent any modifiers 4 spaces (or one tab) lower than the modified view (like the `.cardinalKit(appDelegate)` modifier):
```suggestion
                .sheet(isPresented: isSheetPresented) {
                    OnboardingFlow()
                }
                .interactiveDismissDisabled(true)
                .testingSetup()
```